app/helpers/patient_query_helper.rb

```diff
@@ -485,6 +487,29 @@ def advanced_filter(patients, filters, tz_offset)
                                patients.where_assoc_exists(:laboratories, "laboratories.result = 'positive' AND laboratories.specimen_collection IS NOT NULL")
                              )
                    end
+      when 'assigned-user'
+        if filter[:value].any?
```

Comment:
```suggestion
        if filter[:value].present?
```

same for `jurisdiction`

![Screen Shot 2021-08-10 at 1 17 24 PM](https://user-images.githubusercontent.com/9956561/128905061-68d8d5ae-4070-4569-9d1e-7b0fbae50829.png)

Reply:
Ah that's my bad. I forgot to update handling that after removing the `or` statement from `value`. Thanks, Tim!